src/mlpack/methods/reinforcement_learning/policy/aggregated_policy.hpp

```diff
@@ -0,0 +1,78 @@
+/**
+ * @file aggregated_policy.hpp
+ * @author Shangtong Zhang
+ *
+ * This file is the implementation of AggregatedPolicy class.
+ * An aggregated policy will randomly select a child policy under a given
+ * distribution at each time step.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_RL_POLICY_AGGREGATED_POLICY_HPP
+#define MLPACK_METHODS_RL_POLICY_AGGREGATED_POLICY_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace rl {
+
+/**
+ * @tparam PolicyType The type of the child policy.
+ */
+template <typename PolicyType>
+class AggregatedPolicy
+{
+ public:
+  //! Convenient typedef for action.
+  using ActionType = typename PolicyType::ActionType;
+
+  /**
+   * @param policies Child policies.
+   * @param distribution Probability distribution for each child policy.
+   *   User should make sure its size is same as the number of policies
```

Comment:
Just a minor style issue: "When wrapping a line, the next line should be tabbed twice from where the previous line began." https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping